mycore-base/src/main/java/org/mycore/common/content/transformer/MCRXSLTransformer.java

```diff
@@ -119,6 +119,11 @@ public MCRXSLTransformer() {
         super();
         setTransformerFactory(DEFAULT_FACTORY_CLASS);
     }
+    
+    public MCRXSLTransformer(Class<TransformerFactory> tfClass) {
+        super();
+        setTransformerFactory(tfClass.getName());
+    }
```

Comment:
combine constructors to reuse them, e.g. link empty to this one with (DEFAULT_FACTORY_CLASS) as parameter

Reply:
OK, done!